### PR TITLE
docs: add node/socket debugging information

### PR DESCRIPTION
**Pull request by on3iro**
#### Checklist

- [x] Use a separate branch in your local repo (not `main`).
- [x] Test coverage is 100% (or you have a story for why it's ok). - **just made an addition to the docs ;)**

### CODE REVIEW

File: docs/documentation/debugging.md [hunk 1]
```diff
@@ -91,3 +91,15 @@ Client({
   ),
 })
 ```
+
+### Server + Sockets
+
+The koa-server can be debugged by setting the `DEBUG`-environment before starting it.
```
Comment (delucis) [inline docs/documentation/debugging.md:97]:
```suggestion
The Koa server can be debugged by setting the `DEBUG` environment variable before starting it.
```

File: docs/documentation/debugging.md [hunk 2]
```diff
@@ -91,3 +91,15 @@ Client({
...
+This will give you access to logs of incoming requests as well as the socket.io logs.
+To set the environment simply prepend your npm script to run the server like so:
```
Comment (delucis) [inline docs/documentation/debugging.md:99]:
```suggestion
To set the environment variable, prepend your npm script to run the server like so:
```

**Review by delucis — CHANGES_REQUESTED** [review]
@on3iro great! Thank you for this.

I have a couple of tiny suggestions to use “environment variable” over just “environment” and avoid using “simply” (in case it's not simple for some readers).

**Comment by on3iro** [comment]
> @on3iro great! Thank you for this.
> 
> I have a couple of tiny suggestions to use “environment variable” over just “environment” and avoid using “simply” (in case it's not simple for some readers).

@delucis great suggestions - i'll update my PR accordingly :)

**Review by delucis — APPROVED** [review]

**Comment by delucis** [comment]
Thanks again for this @on3iro!

**Pull request merged**